src/main/java/com/zendesk/maxwell/producer/MaxwellRedisProducer.java

```diff
@@ -31,20 +35,27 @@ public MaxwellRedisProducer(MaxwellContext context) {
 		this.interpolateChannel = channel.contains("%{");
 		this.redisType = context.getConfig().redisType;
 
-		initRedisConnection();
-
-		if (context.getConfig().redisAuth != null) {
-			jedis.auth(context.getConfig().redisAuth);
-		}
-
-		if (context.getConfig().redisDatabase > 0) {
-			jedis.select(context.getConfig().redisDatabase);
+		if (jedisPool == null) {
```

Comment:
I don't think we have to worry about / support the case where the call to `super` has mucked with `jedisPool` (it's deprecated), in other words can probably just eliminate the `null` check here.

Reply:
ok!